src/model/album.rs

```diff
@@ -67,22 +62,117 @@ pub struct FullAlbum {
 }
 
 /// Full Albums
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct FullAlbums {
     pub albums: Vec<FullAlbum>,
 }
 
 /// [link to get list new releases](https://developer.spotify.com/web-api/get-list-new-releases/)
 /// Simplified Albums wrapped by Page object
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
Because some structs can't derive `Eq`, for example, if a struct contains a field and the type of this field if `f32`, this struct can't derive `EQ`, since a `NaN` can't compare with a `NaN`.